tensorflow_lite_support/cc/task/processor/image_preprocessor.cc

```diff
@@ -68,6 +68,22 @@ absl::Status ImagePreprocessor::Init(
         "ImagePreprocessor only supports RGB color space for now.");
   }
 
+  // Determine if the input shape is resizable.
+  const TfLiteIntArray* dims_signature = Tensor()->dims_signature;
+
+  if (dims_signature != nullptr)
+  {
+    const bool height_or_width_mutable = dims_signature->data[1] == -1 || 
+                                         dims_signature->data[2] == -1;
```

Comment:
Nit: merge this line with line 85, such as:

is_input_dynamic_ = dims_signature->data[1] == -1 ||  dims_signature->data[2] == -1;

Reply:
Also, consider to save is_height_mutable_ / is_width_mutable_ instead of is_input_dynamic_.